packages/flutter/test/widgets/widget_inspector_test.dart

```diff
@@ -2066,6 +2068,7 @@ class _TestWidgetInspectorService extends TestWidgetInspectorService {
       expect(event['startTime'], isA<int>());
       data = event['events']! as List<int>;
       newLocations = event['newLocations']! as Map<String, List<int>>;
+      fileLocationsMap = event['locations']! as Map<String, Map<String, List<dynamic>>>;
```

Comment:
you could encode + decode from json in some of these tests to make sure the data is valid JSON. I believe at least one of the tests in this file does that.

Reply:
Good idea! Updated the PR to encode all the events to json when pulling events off the list, in order to verify that there's nothing in there that can't encode.